docs/pages/docs/components/form.mdx

```diff
@@ -63,4 +63,11 @@ Available validation functions:
 - `equals`
 - `contains`
 
+Available combinators:
+
+- `all`: ensures all validators pass
+- `any`: ensures at least one validator passes
+- `none`: ensures no validators pass
+- `compose` (deprecated, use `all`)
```

Comment:
how about marking `compose` as an alias of `all` instead of deprecating it? (`compose` is commonly used in libraries for building validations)

Reply:
Definitely, will do! Can also remove 'all' if you'd prefer that. I'm much more used to parser combinators, that's probably where I got the naming from haha

Reply:
let's keep both, please! 🙏

Reply:
Sounds good! I will be back home from vacation later today and will update this and the other PRs :)